### PR TITLE
Remove screenshare button on mobile composites

### PR DESCRIPTION
**Pull request by JamesBurnside**
# What
Remove screenshot button on mobile call composite and mobile meeting composite

# Why
We want to not show a screen share button when on mobile.

# How Tested
tbd: add screenshots

### CODE REVIEW

**Review by alkwa-msft — APPROVED** [review]

**Review by alcail — APPROVED** [review]

File: packages/react-composites/src/composites/CallComposite/CallScreen.tsx [hunk 1]
```diff
@@ -131,6 +132,18 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
     );
   }
 
+  // Do not show screen share button when composite is optimized for mobile unless the developer
```
Comment (prprabhu-ms) [inline packages/react-composites/src/composites/CallComposite/CallScreen.tsx:135]: This component is already fairly large. Please extract a helper private function:

```

const disableScreehShareIfCallControlVisible = (options: boolean | CallControlOptions | undefined): boolean | CallControlOptions => {
    ...
}
```

File: packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx [hunk 2]
```diff
@@ -19,17 +19,23 @@ export interface MeetingCallControlBarProps {
...
   onChatButtonClicked: () => void;
   onPeopleButtonClicked: () => void;
+  mobileView?: boolean;
 }
 
 /**
  * @private
  */
 export const MeetingCallControlBar = (props: MeetingCallControlBarProps): JSX.Element => {
+  const callControlsOptions: CallControlOptions = {
```
Comment (prprabhu-ms) [inline packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx:29]: Shouldn't the `CallControlOptions` be exposed from both the `CallComposite` and `MeetingComposite`?


Perhaps override the default value for `participantsButton` to `false`, but expose the options still?

**Review by prprabhu-ms — APPROVED** [review]
Overall LGTM.

File: packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx (hunk 2 above)
Comment (JamesBurnside) [reply packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx:29]: They should but they weren't added to meetings when they were added to CallComposite. I'll make a future work item for this

**Pull request merged**